### DISCUSSION TITLE
No new releases of github.com/ory/fosite in one year?

### DISCUSSION THREAD

**Discussion by cfryanr** [ory/.github#83] [General]
Hi,

Would it be possible to create a new release of github.com/ory/fosite? There has not been a new release since one year ago, even though there are several important commits on the main branch, including dependency upgrades.

It would be very helpful for Go projects that use fosite as a dependency if there could be a new release of fosite.

Thanks for your consideration,
Ryan

**Comment by cfryanr** [comment]
cc @aeneasr

**Comment by aeneasr** [comment]
https://github.com/ory/fosite/releases/tag/v0.45.0 :)

**Reply by aeneasr** [reply]
ps: in go you can always use the git hash - no need for using a tagged release: https://stackoverflow.com/questions/53682247/how-to-point-go-module-dependency-in-go-mod-to-a-latest-commit-in-a-repo

**Reply by cfryanr** [reply]
Thanks! Appreciate that!

Yes, that's what I ended up doing. See https://github.com/vmware-tanzu/pinniped/blob/v0.28.0/go.mod#L55.

However, in general, I don't have any way of knowing if there is any partly finished or otherwise "not ready for production use" code on another project's main branch. That's why I prefer to use release tags when possible. As you know, by creating a release, you are signaling to other projects that the release tag is a complete package of enhancements/fixes that are ready to be used. And tools like dependabot will automatically pick up on your signal and let us know that it is time to upgrade our dependency. :)
